pages/morse.js

```diff
@@ -51,7 +51,25 @@ const allResults = (message) => {
 }
 
 const messageToBox = (message) => {
-  return <MorseResultBox label='Základní řešení' message={message} />
+  const inputItems = []
+
+  decode(message).map((msgPart) => {
+    inputItems.push({
+      input: msgPart.string[0],
+      output: {
+        char: (msgPart.char || '').toUpperCase() || null,
+      },
+    })
+    msgPart.string
```

Comment:
For me inputItems.concate seems more readable than `push` inside `map`.

Reply:
I think `map` is wrong anyway, `forEach` would be better.
I am not sure how exactly you imagine the code with `concat`.

Reply:
`forEach` sounds good.